src/test/java/com/easypost/EventTest.java

```diff
@@ -1,47 +1,61 @@
 package com.easypost;
 
-import com.easypost.model.Event;
-import com.easypost.net.EasyPostResource;
-import org.junit.Test;
-
-import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.HashMap;
 import java.util.Map;
 
-import static org.junit.Assert.assertNull;
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.Event;
+import com.easypost.model.EventCollection;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeAll;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class EventTest {
+    private static EventCollection globalEvents;
+    private static Map<String, Object> params = new HashMap<>();
+
+    /**
+     * Setup the testing environment for this file.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @BeforeAll
+    public static void setup() throws EasyPostException{
+        EasyPost.apiKey = System.getenv("EASYPOST_TEST_API_KEY");
+
+        params.put("page_size", Fixture.pageSize());
 
+        globalEvents = Event.all(params);
+    }
+
+    /**
+     * Test retrieving all events.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @Test
+    public void testAll() throws EasyPostException {
+        EventCollection events = Event.all(params);
+
+        assertTrue(events.getEvents().size() <= Fixture.pageSize());
+        assertNotNull(events.getHasMore());
+        for(Event event: events.getEvents()) {
+            assertTrue(event.getId().startsWith("evt_"));
```

Comment:
This should really assert each item is an instance of Event - this is true for all the `all` tests, they should all be adjusted accordingly throughout the suite.